pyvista/utilities/reader.py

```diff
@@ -625,6 +625,48 @@ def set_active_time_value(self, time_value):  # noqa: D102
     def set_active_time_point(self, time_point):  # noqa: D102
         self.reader.SetTimeValue(self.time_point_value(time_point))
 
+    @property
+    def cell_to_point_creation(self):
+        if self.reader.GetCreateCellToPoint():
+            return True
+        return False
+
+    @cell_to_point_creation.setter
+    def cell_to_point_creation(self, value):
+        if value:
+            self.reader.CreateCellToPointOn()
+        else:
+            self.reader.CreateCellToPointOff()
+
+    @property
+    def number_patch_arrays(self):
+        return self.reader.GetNumberOfPatchArrays()
+
+    @property
+    def patch_array_names(self):
+        return [self.reader.GetPatchArrayName(i) for i in range(self.number_patch_arrays)]
+
+    def enable_patch_array(self, name):
+        self.reader.SetPatchArrayStatus(name, 1)
+
+    def disable_patch_array(self, name):
+        self.reader.SetPatchArrayStatus(name, 0)
+
+    def patch_array_status(self, name):
+        if self.reader.GetPatchArrayStatus(name):
+            return True
+        return False
+
+    def enable_all_patch_arrays(self):
+        self.reader.EnableAllPatchArrays()
```

Comment:
By default, only the `internalMesh` is enabled.  We could consider enabling all patches of data, see https://github.com/pyvista/pyvista/discussions/1869 for confusion it can cause.  However, this would also be a deviation from vtk implementation.

Reply:
I also think this makes sense to implement, since it makes it more user-friendly, done in https://github.com/pyvista/pyvista/pull/1896/commits/0b52522e6e57fa180f9c33a753d50b7d4d88d3ac.